src/main/java/de/teamlapen/vampirism/blocks/BloodContainerBlock.java

```diff
@@ -87,34 +95,31 @@ public boolean isNormalCube(BlockState state, IBlockReader worldIn, BlockPos pos
 
     @Override
     public boolean onBlockActivated(BlockState state, World worldIn, BlockPos pos, PlayerEntity playerIn, Hand hand, BlockRayTraceResult hit) {
-        if (!worldIn.isRemote) {
-            FluidUtil.interactWithFluidHandler(playerIn, hand, worldIn, pos, hit.getFace());//TODO 1.14 fluids
-            /*
-            ItemStack stack = playerIn.getHeldItem(hand);
-            if (!stack.isEmpty() && FluidLib.hasFluidItemCap(stack)) {
-                BloodContainerTileEntity bloodContainer = (BloodContainerTileEntity) worldIn.getTileEntity(pos);
-                IFluidHandler source = bloodContainer.getCapability(CapabilityFluidHandler.FLUID_HANDLER_CAPABILITY, null);
-                if (playerIn.isSneaking()) {
-                    FluidActionResult result = FluidUtil.tryFillContainer(stack, source, Integer.MAX_VALUE, playerIn, true);
-                    if (result.isSuccess()) {
-                        playerIn.setHeldItem(hand, result.getResult());
-                    }
-                    //fillContainerFromTank(worldIn, pos, playerIn, stack, bloodContainer);
-                } else {
-                    FluidActionResult result = FluidUtil.tryEmptyContainer(stack, source, Integer.MAX_VALUE, playerIn, true);
-                    //drainContainerIntoTank(worldIn, pos, playerIn, stack, bloodContainer);
-                    if (result.isSuccess()) {
-                        playerIn.setHeldItem(hand, result.getResult());
-                    }
+        FluidUtil.interactWithFluidHandler(playerIn, hand, worldIn, pos, hit.getFace());
```

Comment:
Isn't this line doing something similar to the remaining part?

Reply:
maybe read the whole docs to know, that in creative mode the item doesnt get replaced if emptied